File: DevcampPracticalProjectMocks/js-project-mock/app/controllers/addressController.js
const express = require('express');
const addressController = express.Router();

addressController.get("/:address/balance", (req,res) => {

    const response = require('../mocks/address-balance');

    res.json(response);
    res.end();
      

});

addressController.get("/:address/transactions", (req,res) => {

    const response = require('../mocks/address-transactions');

    res.json(response);
    res.end();

});

module.exports = addressController;

File: DevcampPracticalProjectMocks/c#-project-mock/ProjectMock/ProjectMock/Controllers/AddressController.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Hosting;
using Microsoft.AspNetCore.Mvc;
using Newtonsoft.Json;

namespace ProjectMock.Controllers
{
    [Route("[controller]")]
    public class AddressController : Controller
    {

        IHostingEnvironment _hostingEnvironment;


        public AddressController(IHostingEnvironment hostingEnvironment)
        {
            this._hostingEnvironment = hostingEnvironment;
        }

        [HttpGet("{id}/transactions")]
        public IActionResult Transactions(string id)
        {

            object jsonObject = GetJSON("address-transactions.json");

            return Json(jsonObject);
        }

        [HttpGet("{id}/balance")]
        public IActionResult Balances(string id)
        {

            object jsonObject = GetJSON("address-balance.json");

            return Json(jsonObject);
        }


        public object GetJSON(string name)
        {
            string path = Path.Combine(_hostingEnvironment.WebRootPath, $"mocks/{name}");

            string json = System.IO.File.ReadAllText(path);

            object jsonObject = JsonConvert.DeserializeObject(json);

            return jsonObject;
        }
    }
}